lighthouse-core/computed/metrics/timing-summary.js

```diff
@@ -57,6 +59,9 @@ class TimingSummary {
     const cumulativeLayoutShiftValue = cumulativeLayoutShift &&
       cumulativeLayoutShift.value !== null ?
       cumulativeLayoutShift.value : undefined;
+    const cumulativeLayoutShiftAllFramesValue = cumulativeLayoutShiftAllFrames &&
+      cumulativeLayoutShiftAllFrames.value !== null ?
```

Comment:
can your version ever be null? I don't *think* original CLS could be, but I'm even more certain with the reduce here that this one can't :)